### DISCUSSION TITLE
Create Auth0 user and saving it to mongodb

### DISCUSSION THREAD

**Discussion by hammad-rb** [jowilf/starlette-admin#531] [Q&A] [answered]
Hello,
Is there are way that when I create user it should create in Auth0 and mongodb, is there an example or any say to overwrite?

**Comment by jowilf** [comment]
Take a look at this example -> https://github.com/jowilf/starlette-admin/tree/main/examples/authlib

**Reply by hammad-rb** [reply]
yeah I saw that but its protecting admin by authentication, how could I invite user from logged in admin.

**Reply by jowilf** [reply, marked as answer]
>  I create user it should create in Auth0 and mongodb,

You can use [hooks](https://jowilf.github.io/starlette-admin/tutorial/configurations/modelview/?h=hooks#hooks) to create the user in Auth0 through their API
